generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -318,7 +318,7 @@ _%>
       <div>
       <Row className="justify-content-center">
         <Col md="8">
-          <h2 id="<%= jhiPrefixDashed %>-<%= entityFileName %>-heading">
+          <h2 id="page-heading">
```

Comment:
I would prefer unique ids, can you use the same tempalte in the page objects as well